app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,24 @@
+class ProjectsController < ApplicationController
+  def create
+    @developers = @ptribe.developers
+    total = @ptribe.total 
+    session[:tribe] = []
+    @project = current_contractor.projects.create(project_params.merge({total: total}))
+    @project.developers += @developers 
+    redirect_to contractor_project_path(@project)
```

Comment:
We're going to need to clean this up.  Duh.  Have you ever heard of pushing logic down the stack?